modules/web-console/frontend/app/modules/agent/AgentManager.service.ts

```diff
@@ -373,8 +361,7 @@ export default class AgentManager {
         const oldCluster = _.find(conn.clusters, (cluster) => cluster.id === newCluster.id);
 
         if (oldCluster) {
-            oldCluster.nids = newCluster.nids;
-            oldCluster.addresses = newCluster.addresses;
```

Comment:
@nva can you please explain this change? Did you change what properties are available on cluster object?

Reply:
I added more types and found this error. This changed when we rewrite to java backend and doesn't fixed by @vsisko

Reply:
Good find! 👍